mkdocs/structure/files.py

```diff
@@ -41,10 +43,13 @@ def all(self):
         return True
 
     def is_included(self):
-        return self.value > self.EXCLUDED.value
+        return self.value > self.DRAFT.value
 
     def is_excluded(self):
-        return self.value <= self.EXCLUDED.value
+        return self.value <= self.DRAFT.value
+
+    def is_draft_plus(self):
```

Comment:
Not really happy with this name. I also don't have a suggestion that I really love. But how about `is_included_for_serve`

Reply:
ack.

Went with `is_in_serve` as a shorter version of your suggestion.